docs/tutorial/02-draw-a-snake.md

```diff
@@ -99,47 +99,61 @@ defmodule Snake.Scene.Game do
     viewport = opts[:viewport]
 
     IO.inspect(ViewPort.info(viewport), label: "viewport info")
+
+    {:ok, :ok}
   end
 end
 ```
 
-Running the game again, we see we have some information about the size of the view port in pixels. We can use this information to compute the number of rows and columns in our grid.
+Running the game again, we see we have some information about the size of the viewport in pixels. We can use this information to compute the number of rows and columns in our grid. We'll define the default length of 32 pixels for the edge of one square and then compute how many we can fit onto the screen. As this information is necessary throughout the whole game, we store it in the `state` of the scene.
```

Comment:
I think™ it was part of the feedback in my initial PR for this document that we'll do get rid of the compiler warning **before** continuing on the grid logic. What's your motivation to change it back?

Reply:
I found it confusing that we started to talk about the window size and calculating the number of tiles, then we change the focus to the warning and then again to the calculation

Reply:
Yep, valid point 👍